packages/dev/core/src/Rendering/renderingManager.ts

```diff
@@ -236,6 +236,23 @@ export class RenderingManager {
         }
     }
 
+    /**
+     * Resets the sprites information of the group to prepare a new frame
+     * @internal
+     */
+    public resetSprites(): void {

```

Comment:
Shouldn't you name the function `_resetSprites` if it's an internal usage?

Reply:
they  are all named like this in the file so I kept the consistency :-)